dimod/discrete/cydiscrete_quadratic_model.pyx

```diff
@@ -22,14 +22,10 @@ from libcpp.unordered_set cimport unordered_set
 
 import numpy as np
 
-from dimod.utilities import asintegerarrays, asnumericarrays
-
 
 cdef class cyDiscreteQuadraticModel:
 
     def __init__(self):
-        self.case_starts_.push_back(0)
```

Comment:
The default constructor of the dqm pushes back 0, I think we do not need it here anymore.